package.json

```diff
@@ -1,148 +1,153 @@
+        "box-react-ui": "^22.7.0",
+        "chai": "^4.2.0",
+        "chai-dom": "^1.8.1",
+        "conventional-changelog-cli": "^1.3.5",
+        "conventional-github-releaser": "^2.0.0",
+        "create-react-class": "^15.6.2",
+        "css-loader": "^0.28.7",
+        "cssnano-cli": "^1.0.5",
+        "cypress": "^3.1.5",
+        "eslint": "^4.12.1",
+        "eslint-config-airbnb": "^16.1.0",
+        "eslint-config-prettier": "^2.9.0",
+        "eslint-import-resolver-webpack": "^0.8.3",
+        "eslint-plugin-babel": "^4.1.1",
+        "eslint-plugin-chai-friendly": "^0.4.1",
+        "eslint-plugin-cypress": "^2.2.0",
+        "eslint-plugin-import": "^2.8.0",
+        "eslint-plugin-jsx-a11y": "^6.0.2",
+        "eslint-plugin-react": "^7.5.1",
+        "extract-text-webpack-plugin": "3.0.2",
+        "file-loader": "^1.1.5",
+        "fscreen": "^1.0.2",
+        "husky": "^1.3.1",
+        "i18n-webpack-plugin": "^1.0.0",
+        "jsuri": "^1.3.1",
+        "karma": "^3.1.4",
+        "karma-chai": "^0.1.0",
+        "karma-chai-dom": "^1.1.0",
+        "karma-chrome-launcher": "^2.2.0",
+        "karma-coverage": "^1.1.2",
+        "karma-fixture": "^0.2.6",
+        "karma-html2js-preprocessor": "^1.1.0",
+        "karma-junit-reporter": "^1.2.0",
+        "karma-mocha": "^1.3.0",
+        "karma-mocha-reporter": "^2.2.5",
+        "karma-phantomjs-launcher": "^1.0.4",
+        "karma-sinon": "^1.0.5",
+        "karma-sinon-chai": "^2.0.2",
+        "karma-sourcemap-loader": "^0.3.7",
+        "karma-webpack": "^3.0.5",
+        "lint-staged": "^5.0.0",
+        "lodash": "^4.17.5",
+        "mocha": "^4.0.1",
+        "mock-local-storage": "^1.0.2",
+        "mojito-rb-gen": "^0.0.1",
+        "node-noop": "^1.0.0",
+        "node-sass": "4.9.3",
+        "npm-run-all": "^4.1.5",
+        "optimize-css-assets-webpack-plugin": "^3.2.0",
+        "phantomjs-prebuilt": "^2.1.16",
+        "postcss-loader": "^2.0.9",
+        "postcss-sass": "^0.3.0",
+        "prettier": "^1.8.2",
+        "prettier-eslint-cli": "^4.4.2",
+        "raw-loader": "^0.5.1",
+        "react": "^16.7.0",
+        "react-dom": "^16.7.0",
+        "react-virtualized": "^9.13.0",
+        "sass-loader": "^6.0.6",
+        "sinon": "^7.2.3",
+        "sinon-chai": "3.3.0",
+        "string-replace-loader": "^1.3.0",
+        "style-loader": "^0.19.0",
+        "stylelint": "^8.3.1",
+        "stylelint-config-standard": "^18.0.0",
+        "stylelint-order": "^0.7.0",
+        "stylelint-scss": "^2.2.0",
+        "wait-on": "^3.2.0",
+        "webpack": "^3.10.0",
+        "webpack-bundle-analyzer": "^2.9.1",
+        "webpack-dev-server": "^2.11.3"
+    },
+    "engines": {
+        "node": ">=8.9.4",
+        "yarn": ">=1.10.0"
+    },
+    "scripts": {
+        "build": "yarn setup && yarn clean && yarn build:i18n && yarn lint && yarn build:dev",
+        "build:ci": "yarn setup && yarn webpack --progress --colors --config build/webpack.config.js",
```

Comment:
Do we not need relative paths to webpack & karma anymore?

Reply:
I don't think so. Prepending with `yarn` should force the commands to use local packages.